.github/workflows/presubmit.yml

```diff
@@ -72,26 +71,23 @@ jobs:
         run: |
           cd room && ./gradlew :buildOnServer && cd ../work && ./gradlew :buildOnServer
         id: gradlew_bos
-      - name: Set webhook outputs from ./gradlew
-        if: always()
-        run: |
-          if [ "${{ steps.gradlew_bos.outcome }}" == "success" ] || [ "${{ steps.gradlew_bos.outcome }}" == "skipped" ]
-          then
-            echo "##[set-output name=success;]true"
-          else
-            echo "##[set-output name=success;]false"
-          fi
-        id: gradlew_bos_outputs
       - name: Upload Results
         if: always()
         uses: actions/upload-artifact@v2
         with:
           name: artifacts
           path: ~/dist
-      - name: Completed WebHook
-        if: always()
+      - name: Successful WebHook
+        if: ${{ success() }}
+        uses: tikurahul/github-workflow-webhook-action@master
+        with:
+          url: 'https://androidx.dev/github/androidX/presubmit/hook'
+          secret: ${{ secrets.ANDROIDX_PRESUBMIT_HOOK_SECRET }}
+          payload: '{ "platform": "${{ matrix.os }}", "token": "${{ secrets.GITHUB_TOKEN }}", "state": "completed", "success": true }'
+      - name: Failure WebHook
+        if: ${{ failure() || cancelled() }}
```

Comment:
Are these states based on previous step or the entire job? e.g., what happens if:
```
- name: bOS fails
  ...
- name: always succeeds
  if: always()
  run: true
- name Successful WebHook
  if: success()
  run: echo "this shouldn't run!"
```

Reply:
These are based on the entire job: https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#job-status-check-functions has more information.